docset/windows/appx/Get-NonRemovableAppsPolicy.md

```diff
@@ -0,0 +1,169 @@
+---
+author: romaclac
```

Comment:
```suggestion
author: romaclac
```
It seems that your GitHub Username is rdmaclachlan, on author you should use your GH username.